frontend/__tests__/components/utils/async.spec.tsx

```diff
@@ -95,6 +95,7 @@ describe('AsyncComponent', () => {
     wrapper = mount(<AsyncComponent loader={loader} className={className} />);
   });
 
+  /* TODO: test broke when upgraded to jest 23.x
```

Comment:
`AsyncComponent` is fairly important in our codebase, we shouldn't remove its unit test lightly.

Reply:
I get the following error message which I have a hard time parsing:

![image](https://user-images.githubusercontent.com/12733153/47589653-c963f500-d937-11e8-92bc-af03dd8fbfd9.png)

I can remove the snapshot test and rollback jest version; I had thought it was helpful to update to latest jest and add the ability to do snapshot testing, but if it is something we don't want....